week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,24 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var exchangePrice = price * 1.4 * 1.01;
+  var inUSDFormat = exchangePrice.toLocaleString("en-US", {
+    style: "currency",
+    currency: "USD",
+    minimumFractionDigits: 2
+  });
+  return inUSDFormat;
+}
+function convertToBRL(price) {
+  var exchangePrice = price * 5 * 1.01; // exchange rate is £1 = R$5
+  var inBRLFormat = exchangePrice.toLocaleString("pt-BR", {
+    style: "currency",
+    currency: "BRL",
+    minimumFractionDigits: 2
+  });
+  return inBRLFormat;
+}
```

Comment:
The aim here is to avoid repeating the same logic in both functions (or as much as possible). Imagine that they introduced another 10 foreign currencies, then at an even later date decided to change the fee from 1% to 2%. This would mean having to find all the different places this was done and changing the number, which is error prone. Can you see how you could write things differently so that the transaction fee is only calculated in one place?